### PR TITLE
Vue 3 migration

### PR DESCRIPTION
**Pull request by kelsos**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/defi/airdrops/PoapDeliveryAirdrops.vue [hunk 1]
```diff
@@ -55,7 +61,8 @@ function getImage(event: EventType): string {
 
     <RuiDataTable
       :rows="items"
-      :cols="headers"
+      :cols="
+        headers"
```
Comment (lukicenturi) [inline frontend/app/src/components/defi/airdrops/PoapDeliveryAirdrops.vue:65]: bad formatting i guess

File: frontend/app/src/components/asset-manager/MergeDialog.vue [hunk 2]
```diff
@@ -8,9 +8,9 @@ type Errors = Partial<
   Record<'targetIdentifier' | 'sourceIdentifier', string[]>
 >;
 
-const props = defineProps<{ value: boolean }>();
+const props = defineProps<{ modelValue: boolean }>();
 
-const emit = defineEmits<{ (e: 'input', value: boolean): void }>();
+const emit = defineEmits<{ (e: 'update:model-value', value: boolean): void }>();
 
 const display = useSimpleVModel(props, emit);
```
Comment (lukicenturi) [inline frontend/app/src/components/asset-manager/MergeDialog.vue:15]: defineModel

**Review by lukicenturi — APPROVED** [review]
Other than the comments, look harmless

File: frontend/app/src/components/asset-manager/MergeDialog.vue (hunk 2 above)
Comment (kelsos) [reply frontend/app/src/components/asset-manager/MergeDialog.vue:13]: I did some of them but we can do the rest slowly

**Pull request merged**